### PR TITLE
Div 4218 cleanup 510

### PR DESCRIPTION
**Pull request by tchow8**
[DIV-4218](https://tools.hmcts.net/jira/browse/DIV-4218)

### CODE REVIEW

File: test/end-to-end/codecept.conf.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ function configureChunks() {
 function getTests() {
   console.log('### CONF.preview_env =', CONF.preview_env);  // eslint-disable-line no-console
   if (CONF.preview_env === 'true') {
-    return './paths/**/basicDivorce.js';
+    return './paths/**/*.js';
```
Comment (tchow8) [inline test/end-to-end/codecept.conf.js]: Will revert this when functional all passes

**Comment by tchow8** [comment]
Every single test in functional tests passed.

File: Jenkinsfile_CNP [hunk 2]
```diff
@@ -57,6 +57,8 @@ withPipeline("nodejs", product, component) {
     env.IDAM_API_URL = 'https://idam-api.aat.platform.hmcts.net'
     env.CASE_ORCHESTRATION_SERVICE_DRAFT_URL = 'http://div-cos-aat.service.core-compute-aat.internal/draftsapi/version/1'
     env.PREVIEW_ENV = 'true'
+
+    sh "curl -X PATCH https://idam-api.aat.platform.hmcts.net/testing-support/services/divorce -H 'Accept: application/json' -H 'Content-Type: application/json' -d '[{\"operation\": \"add\", \"field\": \"redirect_uri\", \"value\": \"${env.TEST_URL}/authenticated\"}]'"
```
Comment (hosainnet) [inline Jenkinsfile_CNP]: No longer needed

**Comment by tchow8** [comment]
Reopening with new PR

**Pull request closed without merging**